doc/manual/src/release-notes/rl-2.4.md

```diff
@@ -141,6 +141,8 @@ more than 2800 commits from 195 contributors since release 2.3.
   the evaluation cache. This is made possible by the hermetic
   evaluation model of flakes.
 
+  This is a CLI-level cache and it does not apply additionally to flake `inputs` for example.
+
 * The new `--offline` flag disables substituters and causes all
```

Comment:
Not needed, previous paragraph calls out thst only flake outputs and that it is for commands. O ly confusion may be that getFlake with a select operator is not cached. Plus a very old release note. 

Recommend not removing this change.